bench/src/main.rs

```diff
@@ -154,6 +154,10 @@ struct TxSendData {
     transaction_bytes: u64,
 }
 
+struct ApiCallerResult {
```

Comment:
Dont know why you created a strucutre for one element.

Reply:
returning only one specific and unrelated value from the huge task feels very wrong. i.e. the task is **not** about getting the _bench_start_time_.